amplify/backend/function/createUserNotificationsData/src/utils.js

```diff
@@ -0,0 +1,39 @@
+const fetch = require('cross-fetch');
+
+const TokenType = {
+  COLONY: 'COLONY',
+  ERC20: 'ERC20',
+};
```

Comment:
Is this used anywhere?

Reply:
Good catch, just a left over copy / paste relic 🙃